lib/health_cards/health_card_constraints.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'base64'
+require 'zlib'
+require 'json/minify'
+
+module HealthCards
+  ## Functionality for "Health Cards are Small" section of the Smart Health Cards Specification
+  module HealthCardConstraints
+    def bundle_operations(payload)
+      bundle = payload['vc']['credentialSubject']['fhirBundle']
+      if bundle
+        entries = bundle['entry']
+        entries, url_map = redefine_uris(entries)
+
+        update_elements(entries, url_map)
+      end
+      payload
```

Comment:
This module appears to be modifying the payload that is passed into it. I think that is very undesirable behavior.